packages/auto_mappr/lib/src/extensions/dart_type_extension.dart

```diff
@@ -12,6 +13,18 @@ extension DartTypeExtension on DartType {
       isDartCoreEnum ||
       isDartCoreSymbol;
 
+  /// Is special variant of integer.
+  ///
+  /// See `[Uint8List], [Uint16List], [Uint32List], [Uint64List].`
```

Comment:
Why those backticks?

Reply:
Because otherwise analyser complains that you should `[]` reference only types within library